### ISSUE TITLE
Counter to show number of ads blocked by the app

### ISSUE THREAD

**Issue by PravinBorhade** [TeamNewPipe/NewPipe#4561]
### Checklist

- [x] I checked, but didn't find any duplicates of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

It would be nice to show the number of ads blocked while using the app.

#### Additional context
Lately YT has been pushing ads like crazy, so I think tit would nice to know how many ads has been blocked by the app.

**Comment by TobiGr** [comment]
We don't block ads. We just load the requested stream. Thus counting blocked adds requires to send additional data to YT. I.e. we won't implement that.

**Comment by PravinBorhade** [comment]
@TobiGr thanks for the clarification
